Data/ByteString.hs

```diff
@@ -276,7 +276,7 @@ import GHC.Word hiding (Word8)
 
 -- | /O(1)/ The empty 'ByteString'
 empty :: ByteString
-empty = BS nullForeignPtr 0
+empty = mempty
```

Comment:
Can we remove this definition when we export the one from `D.B.Internal`?

Reply:
Sure. I was hesitant about adding a mostly-redundant export to `D.B.Internal`, but its export list isn't actually as large as I thought it was.